include/cantera/transport/UlnTransport.h

```diff
@@ -0,0 +1,206 @@
+/**
+ *  @file UlnTransport.h
+ *    Headers for the UlnTransport object, which models transport properties
+ *    in ideal gas solutions using a mixture averaged approximation
+ *    (see \ref tranprops and \link Cantera::UlnTransport UlnTransport \endlink) .
+ */
+
+// This file is part of Cantera. See License.txt in the top-level directory or
+// at http://www.cantera.org/license.txt for license and copyright information.
+
+#ifndef CT_ULNTRAN_H
+#define CT_ULNTRAN_H
+
+#include "GasTransport.h"
+#include "cantera/numerics/DenseMatrix.h"
+
+namespace Cantera
+{
+//! Class UlnTransport implements mixture-averaged transport properties for
+//! ideal gas mixtures.
+/*!
+ * The model is based on that described in: R. J. Kee, M. J. Coltrin, and P.
+ * Glarborg, "Chemically Reacting Flow: Theory & Practice", John Wiley & Sons,
+ * 2003.
+ *
+ * The viscosity is computed using the Wilke mixture rule (kg /m /s)
+ *
+ * \f[
+ *     \mu = \sum_k \frac{\mu_k X_k}{\sum_j \Phi_{k,j} X_j}.
+ * \f]
+ *
+ * Here \f$ \mu_k \f$ is the viscosity of pure species \e k, and
+ *
+ * \f[
+ *     \Phi_{k,j} = \frac{\left[1
+ *                  + \sqrt{\left(\frac{\mu_k}{\mu_j}\sqrt{\frac{M_j}{M_k}}\right)}\right]^2}
+ *                  {\sqrt{8}\sqrt{1 + M_k/M_j}}
+ * \f]
+ *
+ * The thermal conductivity is computed from the following mixture rule:
+ * \f[
+ *     \lambda = 0.5 \left( \sum_k X_k \lambda_k  + \frac{1}{\sum_k X_k/\lambda_k} \right)
+ * \f]
+ *
+ * It's used to compute the flux of energy due to a thermal gradient
+ *
+ * \f[
+ *     j_T =  - \lambda  \nabla T
+ * \f]
+ *
+ * The flux of energy has units of energy (kg m2 /s2) per second per area.
+ *
+ * The units of lambda are W / m K which is equivalent to kg m / s^3 K.
+ * @ingroup tranprops
+ */
+class UlnTransport : public GasTransport
```

Comment:
I think the implementation could be greatly simplified if this class inherited `MixTransport` instead.

I would also prefer a more descriptive class name, e.g. `UnityLewisNumberTransport` (and in general, I think "*unity* Lewis number" is the preferred phrase, not *unitary*.